index.html

```diff
@@ -41,10 +41,10 @@ <h3 class="connect">LET’S CONNECT</h3>
 				<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img
 							src='images/linkedin.svg' width='20' height='20' alt="LinkedIn Profile" title=''></a>
 				</li>
-				<li class="sm"><a href=""><img src='images/m.svg' width='20' height='20' alt=''></a></li>
+				<li class="sm"><a href=""><img src='images/m.svg' width='20' height='20' alt='Medium Profile'></a></li>
 				<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg' width='20'
 							height='20' alt="GitHub Profile" title=''></a></li>
-				<li class="sm"><a href=""><img src='images/v.svg' width='20' height='20' alt=''></a></li>
+				<li class="sm"><a href=""><img src='images/v.svg' width='20' height='20' alt='Angellist Profile'></a></li>
```

Comment:
Social icons medium and angellist do not have alt text on lines 44 and 47. Corrected.